android/build.gradle

```diff
@@ -48,4 +48,5 @@ dependencies {
     implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
     implementation 'com.facebook.react:react-native:+' // From node_modules
     api 'io.branch.sdk.android:library:5.12.4'
+    implementation 'com.google.android.gms:play-services-base:18.5.0'
```

Comment:
Does this need to be distributed?

Reply:
I just removed it and the IV code still worked, so I committed the removal as we don't seem to need it. 

Thanks Gabe, good catch!